app/forms/questionnaire_form.py

```diff
@@ -149,23 +166,39 @@ def validate_mutually_exclusive_question(self, question):
             return False
         return True
 
-    def _get_target_total_and_currency(self, calculation, question):
+    def _get_target_total_and_currency(
+        self,
+        calculation: Calculation,
+        question: QuestionSchema,
+    ) -> Union[tuple[Union[Calculation, AnswerValueTypes], Optional[str]], None]:
```

Comment:
Not a blocker, this is one type, i.e `Optional[tuple[...]]`. Don't think one is necessarily any more readable IMO, therefore just following the style guide.
```suggestion
    ) -> Optional[tuple[Union[Calculation, AnswerValueTypes], Optional[str]]]:
```